controls/RHEL-07-040190.rb

```diff
@@ -31,8 +31,8 @@
 If “ClientAliveInterval” is not set to “600” in /etc/ ssh/sshd_config, and a lower value is not documented with the Information System Security Officer (ISSO) as an operational requirement, this is a finding.'
 
 # START_DESCRIBE RHEL-07-040190
-  describe file('') do
-    it { should match // }
+  describe sshd_config do
+    its('ClientAliveInterval') { should eq '600' }
```

Comment:
Here is a diff that you can take a look.
```
diff --git a/controls/RHEL-07-040190.rb b/controls/RHEL-07-040190.rb
index 13b6d4c..2746212 100644
--- a/controls/RHEL-07-040190.rb
+++ b/controls/RHEL-07-040190.rb
@@ -4,6 +4,9 @@
 # date: 2016-01-14
 # description: This Security Technical Implementation Guide is published as a tool to improve the security of Department of Defense (DoD) information systems. The requi
 # impacts
+
+timeout_value = attribute('client_alive_interval', default: '600', description: 'value for the ClientAliveInterval')
+
 title 'RHEL-07-040190 - All network connections associated with SSH traffic must terminate at the end of the session or after 10 minutes of inactivity, except to fulfil
 control 'RHEL-07-040190' do
   impact 0.5
@@ -30,9 +33,10 @@ ClientAliveInterval 600
 
 If “ClientAliveInterval” is not set to “600” in /etc/ ssh/sshd_config, and a lower value is not documented with the Information System Security Officer (ISSO) as an ope
 
+
 # START_DESCRIBE RHEL-07-040190
   describe sshd_config do
-    its('ClientAliveInterval') { should eq '600' }
+    its('ClientAliveInterval') { should eq timeout_value }
   end
 # STOP_DESCRIBE RHEL-07-040190
```
The way to change the timeout_value would be to use an attributes.yml file to override the attribute value.

Here are the contents of my attributes.yml
`client_alive_interval: '900'`

And how I can use this is 
`inspec exec inspec-stig-rhel7/controls/RHEL-07-040190.rb --attrs inspec-stig-rhel7/attributes/attributes.yml `